pkg/get/tools.go

```diff
@@ -1134,5 +1134,33 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/{{.Name}}_{{.Version}}_{{$osString}}_{{$arch}}{{$ext}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:   "FairwindsOps",
+			Repo:    "polaris",
+			Name:    "polaris",
+			Version: "3.2.1",
+			BinaryTemplate: `
+				{{$arch := "386"}}
```

Comment:
I'd suggest removing `386` and defaulting to `x86_64`.

I can't imagine anyone using 386 and we have no support in any other apps or CLIs, so don't want to introduce it.